config/locales/en.yml

```diff
@@ -433,7 +433,7 @@ en:
       heading: "No entry with the id: %{id}"
       body: "Sorry, there is no diary entry or comment with the id %{id}. Please check your spelling, or maybe the link you clicked is wrong."
     diary_entry:
-      posted_by_html: "Posted by %{link_user} on %{created} in %{language_link}"
+      posted_by_html: "Posted by %{link_user} on %{created} in %{language_link} last updated on %{updated}"
```

Comment:
Unfortunately we can't change the number of interpolation variables in an existing string. Doing so causes server errors until all the translation strings are updated. So if you change the number of interpolation variables, you have to also rename the translation key (`posted_by_html`). 

Alternatively, you could look at adding this as a separate sentence, i.e. adding a full stop to the existing sentence, and starting a new one with the last updated information.